app/src/main/res/values-en-rGB/strings.xml

```diff
@@ -0,0 +1,23 @@
+<resources>
+
+    <string name="title_favourites">Favourites</string>
+    <string name="notification_favourite_format">%s favourited your toot</string>
+    <string name="action_favourite">Favourite</string>
+    <string name="action_unfavourite">Remove favourite</string>
+    <string name="action_view_favourites">Favourites</string>
+    <string name="action_open_faved_by">Show favourites</string>
+    <string name="pref_title_notification_filter_favourites">my posts are favourited</string>
+    <string name="notification_favourite_name">Favourites</string>
+    <string name="notification_favourite_description">Notifications when your toots get marked as favourite</string>
+
+    <string name="description_status_favourited">Favourited</string>
+
+    <string name="error_authorization_unknown">An unidentified authorisation error occurred.</string>
+    <string name="error_authorization_denied">Authorisation was denied.</string>
```

Comment:
My dictionary says in BE both authorisation and authorization are ok, but not if one of them is preferable.

Reply:
'authorisation' is preferred in BE :) 's' over 'z' in these kinds of situations, generally.